index.html

```diff
@@ -48,78 +57,127 @@ <h2>Tonic</h2>
-                    <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+            <div class="deskImg3"></div>
+            <div class="grid-colum">
+                <h4>Tonic</h4>
+                <h4 class="deskTitle1">Facebook 360</h4>
+                <ul class="mobile">
+                    <li class="company" id="deskHideCanopy">CANOPY</li>
+                    <li class="deskCompany2">FACEBOOK</li>
+                    <li id="deskHideJob">Back End Dev</li>
+                    <li class="mobileHideJob">Full Stack Dev</li>
+                    <li>2013</li>
                 </ul>
+                <p class="mobileP">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="deskP">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
+                <div class="mobileTeck">
+                    <ul class="teck-list">
+                        <li><button class="mobile-btn" type="submit">HTML</button></li>
+                        <li><button class="mobile-btn" type="submit">CSS</button></li>
+                        <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                    </ul>
+                </div>
+                <div class="deskTeck">
+                    <ul class="deskTeck-list">
+                        <li><button class="desk-btn" type="submit">HTML</button></li>
+                        <li><button class="desk-btn" type="submit">Ruby on Rails</button></li>
+                        <li><button class="desk-btn" type="submit">CSS</button></li>
+                        <li><button class="desk-btn" type="submit">JavaScript</button></li>
+                    </ul>
+                </div>
+                <button class="see-mobile-btn" type="submit">See Projects</button>
             </div>
-            <button class="see-mobile-btn" type="submit">See Projects</button>
         </section>
         <!-- card4 -->
         <section class="card4">
             <div class="mobileimage4"></div>
-            <h3>Multi-Post <br> Stories</h3>
-            <ul class="mobile">
-                <li class="company">CANOPY</li>
-                <li>Back End Dev</li>
-                <li>2013</li>
-            </ul>
-            <p class="mobile-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <div class="teck">
-                <ul class="teck-list">
-                    <li><button class="mobile-btn" type="submit">HTML</button></li>
-                    <li><button class="mobile-btn" type="submit">CSS</button></li>
-                    <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+            <div class="deskImg4"></div>
+            <div class="grid-colum">
+                <h3>Multi-Post <br> Stories</h3>
+                <h3 class="deskTitle1">Uber Navigation</h3>
```

Comment:
Hi @efmomoh,

I've reviewed your project, and I have a few suggestions for you to improve the look of your project design.

- [ ] **font-weight**: The previous reviewer suggested that the font of the card text should not be bold. I agree with this suggestion. The bold font is too heavy for the card text. I would recommend using a regular font weight just like the [provided design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&mode=design&t=RCQz3unX3es0yTQS-0).

- [ ] **Margin**: The previous reviewer also suggested that you add a margin left to the card text. This is a good suggestion, as it will give the text a little bit of padding from the left side of the card. I would recommend a margin of 10px.

Figma design
-----
<img width="474" alt="Screenshot 2023-07-15 at 7 20 43 PM" src="https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/44798044/f38a5870-c001-4da5-9eba-7739acfbdc95">

Your Implemented design
-------
<img width="1253" alt="Screenshot 2023-07-15 at 7 08 56 PM" src="https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/44798044/31df4674-8bcf-4b6d-88c1-bd8766dbc7d6">